app/tests/integration/nimbus/test_e2e_create_experiment.py

```diff
@@ -228,3 +228,62 @@ def test_create_new_experiment_remote_settings_reject(selenium, base_url):
             selenium.refresh()
     else:
         raise AssertionError("Experiment page didn't load")
+
+
+def test_create_new_experiment_remote_settings_timeout(selenium, base_url):
+    experiment_name = f"name here remote {random.randint(0, 1000)}"
+
+    selenium.get(base_url)
+    home = HomePage(selenium, base_url).wait_for_page_to_load()
+    # home.tabs[-1].click()  # Click drafts
+    # current_experiments = None
+    # try:
+    #     current_experiments = len(home.tables[0].experiments)
+    # except TimeoutException:
+    #     current_experiments = 0
+    experiment = home.create_new_button()
+    experiment.public_name = experiment_name
+    experiment.hypothesis = "smart stuff here"
+    experiment.application = "DESKTOP"
+
+    # Fill Overview Page
+    overview = experiment.save_and_continue()
+    overview.public_description = "description stuff"
+    overview.select_risk_brand_false()
+    overview.select_risk_revenue_false()
+    overview.select_risk_partner_false()
+
+    # Fill Branches page
+    branches = overview.save_and_continue()
+    branches.remove_branch()
+    branches.reference_branch_name = "name 1"
+    branches.reference_branch_description = "a nice experiment"
+    branches.feature_config = "No Feature Firefox Desktop"
+
+    # Fill Metrics page
+    metrics = branches.save_and_continue()
+
+    # Fill Audience page
+    audience = metrics.save_and_continue()
+    audience.channel = "Nightly"
+    audience.min_version = 80
+    audience.targeting = "US_ONLY"
+    audience.percentage = 50.0
+    audience.expected_clients = 50
+    audience.save_btn()
+    review = audience.save_and_continue()
+
+    # Review and approve
+    selenium.find_element_by_css_selector("#PageRequestReview")
+    review.launch_without_preview.click()
+    review.request_review.click_launch_checkboxes()
+    review.request_review.request_launch_button.click()
+    review.approve()
+    for attempt in range(35):
+        try:
+            review.timeout_text.is_displayed()
```

Comment:
How... does this work?
Is the timeout window just a very small number in CI (enough that we are reasonably confident we'd hit it in 35 attempts with a 2s delay; or I guess ~70 seconds)?
If we ran the tests locally or on stage, would the test fail if the review timeout is like 5-10 minutes?

Reply:
The worker should run every 30 seconds, so I am waiting for it to push to kinto and then timeout. So I need a little over a minute, but I should probably add more time.

Reply:
Yeah, might be safer to either increase the range stop value, or possibly the sleep interval.
50 loops @ 2s pause = 100s (1m:40s)
35 loops @ 3s pause = 105s (1m:45s)